plugins/services/src/js/components/modals/ServiceDestroyModal.js

```diff
@@ -195,4 +288,4 @@ ServiceDestroyModal.propTypes = {
   ]).isRequired
 };
 
-module.exports = ServiceDestroyModal;
+module.exports = injectIntl(ServiceDestroyModal);
```

Comment:
Let's please follow the examples on https://github.com/yahoo/react-intl/wiki/API#injectintl and define a prop type (`intl: intlShape.isRequired`) when using the injectIntl HOC to properly document how we're providing the functionality.